pkg/upgrade/upgrade.go

```diff
@@ -65,6 +94,66 @@ func TryUpgrade(cli *clientv3.Client) error {
 
 // upgradeToVer1 does upgrade operations from Ver0 to Ver1.
 // in fact, this do nothing now, and just for demonstration.
-func upgradeToVer1(cli *clientv3.Client) error {
+func upgradeToVer1(cli *clientv3.Client, uctx Context) error {
+	return nil
+}
+
+// upgradeToVer2 does upgrade operations from Ver1 to Ver2 (v2.0.0-rc3) to upgrade syncer checkpoint schema
```

Comment:
```suggestion
// upgradeToVer2 does upgrade operations from Ver1 to Ver2 (v2.0.0-rc.3) to upgrade syncer checkpoint schema
```

maybe we release `v2.0.0` GA next time? 😄

Reply:
https://github.com/pingcap/dm/pull/988/commits/556ffaf48bbaeb9e8868e573908f810bfadb37fb